salt/php-fpm/files/default.conf

```diff
@@ -6,3 +6,5 @@ listen.owner = {{ listen_user }}
 listen.group = {{ listen_user }}
 pm = static
 pm.max_children = 10
+php_admin_flag[log_errors] = on
```

Comment:
I have coded error logs to be always on.
Is there a situation we may want to disable them?

Reply:
I can't think of one :)